codespell_lib/data/dictionary.txt

```diff
@@ -597,20 +729,28 @@ announcments->announcements
 annualy->annually
 annuled->annulled
 anohter->another
+anologon->analogon
 anomolies->anomalies
 anomolous->anomalous
 anomoly->anomaly
 anonimity->anonymity
+anonymus->anonymous
+anormaly->abnormally
 anounced->announced
 anouncement->announcement
+anroid->android
 ans->and
 ansalisation->nasalisation
 ansalization->nasalization
 ansestors->ancestors
+ansynchronous->asynchronous
+antaliasing->antialiasing
 antartic->antarctic
+anthor->another
 anthromorphization->anthropomorphization
 anthropolgist->anthropologist
 anthropolgy->anthropology
+antialialised->antialialiased
```

Comment:
Antialialiased isn't a word, it should be antialiased.

Reply:
Good catch. Fixed